nova.ini

```diff
@@ -0,0 +1,45 @@
+# Ansible OpenStack external inventory script
```

Comment:
I'd prefer we not include an ini file that the user is expected to update...  maybe add nova.ini to the .gitignore?